style.css

```diff
@@ -392,61 +401,83 @@ body {
   padding-top: 10px;
 }
 
-.image-container {
+.partners1 {
+  background-color: #272a31;
+  color: #dedede;
+  padding-bottom: 20px;
+}
+
+.partners1 h3 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
- [ ] Please remove the partner and footer sections from the homepage mobile view as suggested earlier. I understand your concept to be dynamic, but all the same let's try and keep to the given template. 👍